docs/build-macos.md

```diff
@@ -19,7 +19,7 @@ installed and the license agreed to:
 
 ## Build using Homebrew
 
-1. Download and install Homebrew: <https://brew.sh>
+1. Download and install Homebrew: <https://brew.sh>.
 2. Follow instructions in [README.md](/README.md).
 
```

Comment:
User comes here for steps on installing X-code, then back to README for installation macOS packages, then that points to BUILD for meson commands.

Let's snip that middle step and include the dependencies here as step 2:

brew install ccache meson libpng sdl2 sdl2_net opusfile fluid-synth

Then step 3 point them to BUILD.md for the meson commands.

Reply:
> In the spirit of (possible) cleanup - some sections of Meson's output feel too verbose (and redundant):
> 
> ![2021-02-18_10-33](https://user-images.githubusercontent.com/1557255/108404741-10471a00-71d5-11eb-8f72-06108bb579cf.png)
> 
> Ideally we'd set a comfortable verbosity level in our `meson.build`, but I couldn't figure out how to adjust it.
> 
> I have no problem with meson reporting `PKG_CONFIG` once, but I don't need it driven into my eyeballs line-after-line! If we can't dial this back, then I'll open a ticket over at meson.

Well, that PKG_CONFIG warning is there to warn about something… most likely that you are inheriting PKG_CONFIG environment variable, and you shouldn't (do you need it in your everyday builds?). If you need it (which I find unlikely) - does it disappear after you pass `-Dpkg_config_path='<your-paths-if-really-really-needed>' to `meson setup`? BTW, same goes for CXXFLAGS, CPPFLAGS, etc. Meson is already smart about picking those.

As for yellow warnings about minimum meson version… The only way to get rid of that is by rising the minimum version… or by removing "not found" dependency messages… I think we would consider making the minimum meson version higher once again… perhaps to 0.51.0…